Source/WebCore/style/RuleSet.h

```diff
@@ -118,15 +120,20 @@ class RuleSet : public RefCounted<RuleSet> {
     bool hasContainerQueries() const { return !m_containerQueries.isEmpty(); }
     Vector<const CQ::ContainerQuery*> containerQueriesFor(const RuleData&) const;
 
+    bool hasScopeQueries() const { return !m_scope.isEmpty(); }
```

Comment:
I don't think "scope query" (unlike "container query") is a spec term.  Maybe `hasScopes()` or `hasScopedRules()`?

Reply:
What about `hasScopeRules` ?